package.json

```diff
@@ -1,6 +1,7 @@
 {
   "private": true,
   "scripts": {
+    "wasm:build": "cargo build -p spl-token-2022 --target=\"wasm32-unknown-unknown\"",
```

Comment:
This is nit, but for consistency, can we rename this to maybe `programs:build-wasm` since we are specifically building the program here. Also, do you think we can add a script in `scripts.rust/build-wasm` like https://github.com/solana-program/token-2022/blob/main/scripts/rust/build-sbf.mjs for wasm build and execute that script here instead? That way we can re-use the same script for multiple programs in the repo maybe in the future.

Reply:
All done.